kitchen.yml

```diff
@@ -22,6 +22,8 @@ suites:
     run_list:
       - recipe[osl-nginx::default]
       - recipe[osl-nginx::mon]
+    excludes:
+      - centos-8
```

Comment:
We still need parts of the mon recipe to work with CentOS 8. I recommend you just put an if block around the munin parts to exclude CentOS 8.

Reply:
OKay sounds good, will do

Reply:
Doesn't look like you did this?

Reply:
oops sorry I forgot to leave a comment. So the packages I included from nrpe are because the nrpe recipe breaks on c8 because it sets the versions to nil. The only line in mon that doesnt reference munin is the nrpe include, so I just kept it how it was

Reply:
I think the issue you're running into with NRPE are related to the EPEL issues we've been having. I recommend you just run a second converge and it should be fine. Basically you should add the following in the `mon` recipe after including the nrpe recipe:
```ruby
if node['platform_version'].to_i < 8
 #munin stuff
```

You'll need to update some tests to account for that however.